CHANGELOG.rst

```diff
@@ -32,6 +32,9 @@ Deprecated
 
 Fixed
 ^^^^^
+* `@senwu`_: Fix pdf_path cannot be without a trailing slash.
+  (`#442 <https://github.com/HazyResearch/fonduer/issues/442>`_)
+  (`#457 <https://github.com/HazyResearch/fonduer/pull/457>`_)
```

Comment:
Minor, but I just found this line should be "(`#459 <https://github.com/HazyResearch/fonduer/pull/459>`_)".